src/bot/mtproto-api/bot/scripts/sms-auth/setup-env.ts

```diff
@@ -17,21 +17,21 @@ dotenv.config();
  */
 
 class SetUpHandler {
-  private _env: Context["env"] = {
-    GITHUB_PAT_TOKEN: "",
+  private _env = {
+    REPO_ADMIN_ACCESS_TOKEN: null,
     TELEGRAM_BOT_ENV: {
       botSettings: {
         TELEGRAM_BOT_ADMINS: [],
-        TELEGRAM_BOT_TOKEN: "",
-        TELEGRAM_BOT_WEBHOOK: "",
-        TELEGRAM_BOT_WEBHOOK_SECRET: "",
+        TELEGRAM_BOT_TOKEN: null,
+        TELEGRAM_BOT_WEBHOOK: null,
+        TELEGRAM_BOT_WEBHOOK_SECRET: null,
       },
       mtProtoSettings: {
-        TELEGRAM_API_HASH: "",
+        TELEGRAM_API_HASH: null,
         TELEGRAM_APP_ID: 0,
       },
     },
-  };
+  } as unknown as Context["env"];
```

Comment:
Seems dangerous to force this casting.

Reply:
I need to force the cast because I've had to type things as `null` in here when `Context["env"]` would never allow these values to be `null` or `undefined` or Typebox would cry on entry.